src/components/environment-map.js

```diff
@@ -0,0 +1,43 @@
+import cubeMapPosX from "../assets/images/cubemap/posx.jpg";
+import cubeMapNegX from "../assets/images/cubemap/negx.jpg";
+import cubeMapPosY from "../assets/images/cubemap/posy.jpg";
+import cubeMapNegY from "../assets/images/cubemap/negy.jpg";
+import cubeMapPosZ from "../assets/images/cubemap/posz.jpg";
+import cubeMapNegZ from "../assets/images/cubemap/negz.jpg";
```

Comment:
I personally dislike this style, because you don't usually look at imports and think "oh, right here is where I'm downloading N megabytes of stuff." I would rather make explicit fetches for them in `loadEnvMap`.